analytic_operating_unit/views/analytic_account_view.xml

```diff
@@ -11,6 +11,7 @@
                 <field
                     name="operating_unit_ids"
                     widget="many2many_tags"
+                    domain="[('user_ids', '=', uid)]"
```

Comment:
Oh well, if the module doesn't work with this line, then remove it :S I just thought that it was needed as it was solved this way in v12.

Reply:
I confirm that fix do not work in v15. Can you remove that part?